lang/en.json

```diff
@@ -553,6 +553,58 @@
         "WindowUnloadTriggerName": "Window Unload",
         "ZendeskChatTagDescription": "Reach your customers, instantly via web, mobile, and messaging.",
         "ZendeskChatTagHelp": "This tag allows you to add the Zendesk Chat form to your website.",
-        "ZendeskChatTagName": "Zendesk Chat (formerly Zopim)"
+        "ZendeskChatTagName": "Zendesk Chat (formerly Zopim)",
```

Comment:
It would be great if you could keep the en.json ordered as it makes diffs in the future nicer.

If you don't want to do it manually as that's annoying, you can use something like https://github.com/Findus23/matomo-utils/blob/main/sortTranslationKeys.py (comment out the lines that ignore plugins not by me)

Reply:
Thinking about it, we could even create a new console command that sorts all translation keys (ideally in the same method python uses)

Reply:
@Findus23 feel free to create an issue for that and we can schedule it but be great to not work on it for now and having it sorted might not be that important. For example it can also be beneficial to have translation keys grouped by logic etc.

Reply:
Updated the en.json file.